src/open_inwoner/accounts/tests/test_auth.py

```diff
@@ -1253,6 +1392,7 @@ def test_page_show_config_text(self):
         self.assertContains(response, "Hello registration text")
         self.assertContains(response, ' href="http://foo.bar/" ')
 
+    @skip("Debug")
```

Comment:
Should be removed probably?

Reply:
Fixed this, the `NecessaryFieldsMiddleware` was essentially disabled due to this change: https://github.com/maykinmedia/open-inwoner/pull/884/commits/68ca2fc755dcc3d145ceabfd1abe283bfd9c90e3#diff-09a70cba3ae30157873d6826d98ae3cf634d9f0c764558ef37e98b4620443217R18. I removed that line and switched the order of the `KvKLoginMiddleware` and `NecessaryFieldsMiddleware` to fix other tests